torchrl/envs/common.py

```diff
@@ -516,7 +516,7 @@ def append_transform(
         self,
         transform: "Transform"  # noqa: F821
         | Callable[[TensorDictBase], TensorDictBase],
-    ) -> None:
+    ) -> EnvBase:
```

Comment:
This should actually return `TransformedEnv`.

However, I'm guessing we don't want to import it at the top to prevent loading a lot of modules that won't be accessed?

Reply:
Either `EnvBase` or "torchrl.envs.TransformedEnv" but we can't import it 
`EnvBase` could be the best compromise!